docker-compose.yml

```diff
@@ -6,8 +6,27 @@ services:
       - 8080:80
     volumes:
       - ./nginxconf/default.conf:/etc/nginx/conf.d/default.conf
-      - ./nginxconf/nginx.conf:/usr/local/openresty/nginx/conf/nginx.conf
+      - ./nginxconf/nginx.conf:/usr/local/openresty/nginx/conf/nginx.conf    
+    networks:
+      frontend:
+        ipv4_address: 172.20.0.2
```

Comment:
this caused issues for me because:
```
Creating network "nginx-test-rig_frontend" with the default driver
ERROR: Pool overlaps with other one on this address space
```
In general using hard-coded address space isn't a good idea for a shareable composition. 
Please remove addresses and allow docker daemon to choose available ones.